### ISSUE TITLE
Question about mongoose-detective

### ISSUE THREAD

**Issue by OliverFischer** [florianholzapfel/express-restify-mongoose#424]
Hi @Zertz ,

by updating my project to the actual master npm complains about the peer-dependency in the mongoose-detective subproject. 

```
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: mongoose@5.11.18
npm WARN node_modules/mongoose
npm WARN   mongoose@"5.11.18" from the root project
npm WARN   1 more (express-restify-mongoose)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer mongoose@"^4.0.0" from mongoose-detective@1.0.0
npm WARN node_modules/express-restify-mongoose/node_modules/mongoose-detective
npm WARN   mongoose-detective@"~1.0.0" from express-restify-mongoose@6.1.2
npm WARN   node_modules/express-restify-mongoose
```

Do you plan to update this project or should I really don't care about it :-) ?

**Comment by Zertz** [comment]
In this case it doesn't matter because it was actually used as a dev dependency. I fixed it in the latest [mongoose-detective](https://github.com/Zertz/mongoose-detective) release and updated other dependencies as well.

Can you submit a PR to upgrade the package in this project?

**Comment by OliverFischer** [comment]
Sure. Did it.

**Comment by Zertz** [comment]
Closed by #425
